data/ci/jenkins/config/config.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<hudson>
+  <disabledAdministrativeMonitors>
+    <string>hudson.diagnosis.ReverseProxySetupMonitor</string>
+    <string>OldData</string>
+  </disabledAdministrativeMonitors>
+  <version>2.60.2</version>
+  <numExecutors>2</numExecutors>
+  <mode>NORMAL</mode>
+  <useSecurity>true</useSecurity>
+  <authorizationStrategy class="org.jenkinsci.plugins.GithubAuthorizationStrategy" plugin="github-oauth@0.27">
```

Comment:
I'm guessing that `0.27` is a version number? What is our strategy to ensure we keep these plugins current to minimise any potential security impact?